nautobot_circuit_maintenance/__init__.py

```diff
@@ -75,8 +76,7 @@ class CircuitMaintenanceConfig(NautobotAppConfig):
     verbose_name = "Circuit Maintenance Management"
     version = __version__
     author = "Network to Code, LLC"
-    author_email = "opensource@networktocode.com"
-    description = "Nautobot App that automatically manages network circuit maintenance notifications. Dynamically reads email inboxes (or APIs) and updates Nautobot mapping circuit maintenances to devices."
+    description = "Nautobot app to automatically handle Circuit Maintenances Notifications."
```

Comment:
Previous description seems preferable?

Reply:
Yes, looks like the description was updated at one point, but the change was not made to the `.cookiecutter.json`